### PR TITLE
Edits to Kasten by Veeam guide

### PR DESCRIPTION
**Pull request by ebgitelman**

### CODE REVIEW

File: advocacy_docs/partner_docs/KastenbyVeeam/04-ConfiguringVeeamKasten.mdx [hunk 1]
```diff
@@ -16,32 +16,32 @@ Implementing EDB Postgres for Kubernetes with Kasten by Veeam requires the follo
 - Kasten K10 installed on your system
 
 !!! Note
-    For this integration, use the **example.yaml** files provided in each section for the appropriate Kasten configuration pieces, and change any environment variables per your specific needs.
+    For this integration, use the `example.yaml` files provided for the appropriate Kasten configuration pieces, and change any environment variables per your specific needs.
     
-    The **Add the Backup Decorator Annotations to the Cluster** section is the important section for the Kasten addon integration. 
+    [Add the backup decorator annotations to the cluster](#add-the-backup-decorator-annotations-to-the-cluster) is important for the Kasten add-on integration.
```
Comment (dwicinas) [inline advocacy_docs/partner_docs/KastenbyVeeam/04-ConfiguringVeeamKasten.mdx:21]: This edit seemed a little less clear than the original text, even though that text is somewhat awkwardly worded.

**Review by dwicinas — APPROVED** [review]
I made one comment. It's not super important, but you might give it a little thought. Otherwise, it loooked good.

File: advocacy_docs/partner_docs/KastenbyVeeam/04-ConfiguringVeeamKasten.mdx (hunk 1 above)
Comment (ebgitelman) [reply advocacy_docs/partner_docs/KastenbyVeeam/04-ConfiguringVeeamKasten.mdx]: Agreed. Turned it into a proper reference. Thank you!

**Review by josh-heyer — APPROVED** [review]
Looks great!

**Pull request merged**